### DISCUSSION TITLE
The nuget package ModernWpf runtime is NET5,Why can ReplayBook (net6) run。

### DISCUSSION THREAD

**Discussion by QQ2287991080** [fraxiinus/ReplayBook#188] [Troubleshooting]
.

**Comment by fraxiinus** [comment]
I just had to specify the version:

https://github.com/Kinnara/ModernWpf/issues/307#issuecomment-796309409

https://github.com/fraxiinus/ReplayBook/blob/8f326e6b14fc3d2bef81c29114c03b8fb6097574/src/UI.Main/UI.Main.csproj#L3

**Reply by QQ2287991080** [reply]
omg ,thanks.
